core/arch/arm/kernel/kern.ld.S

```diff
@@ -394,6 +394,12 @@ SECTIONS
 	__init_size = __data_end - CFG_TEE_LOAD_ADDR;
 	__init_mem_usage = __end - CFG_TEE_LOAD_ADDR;
 #endif
+
+#ifdef CFG_PLAT_EXTRA_LD_SCRIPT
```

Comment:
How about include plat.ld.S in core/arch/arm/plat-rockchip/kern.ld.S, like this: 
#include "../kernel/kern.ld.S"
#include "plat.ld.S"